src/main/java/build/buildfarm/worker/Worker.java

```diff
@@ -486,17 +489,30 @@ public FileVisitResult postVisitDirectory(Path dir, IOException exc) throws IOEx
     return latin1;
   }
 
-  private static WorkerConfig toWorkerConfig(InputStream inputStream) throws IOException {
+  private static WorkerConfig toWorkerConfig(InputStream inputStream, WorkerOptions options) throws IOException {
     WorkerConfig.Builder builder = WorkerConfig.newBuilder();
     String data = new String(convertFromLatin1(ByteStreams.toByteArray(inputStream)));
     TextFormat.merge(data, builder);
     return builder.build();
   }
 
+  private static void printUsage(OptionsParser parser) {
+    System.out.println("Usage: CONFIG_PATH");
+    System.out.println(parser.describeOptions(Collections.<String, String>emptyMap(),
+                                              OptionsParser.HelpVerbosity.LONG));
+  }
+
   public static void main(String[] args) throws Exception {
-    Path configPath = Paths.get(args[0]);
+    OptionsParser parser = OptionsParser.newOptionsParser(WorkerOptions.class);
+    parser.parseAndExitUponError(args);
+    List<String> residue = parser.getResidue();
+    if (residue.isEmpty()) {
+      printUsage(parser);
+      return;
```

Comment:
This will exit the server with a good status, likely not a desirable result if a config isn't specified.

Reply:
Well that is IMO a matter of philosophy. Generally I do not care enough either way. So what behavior do you want exactly?

Reply:
I want the worker to exit with a bad status when the config argument is omitted.

Reply:
So throw an exception here!? Which?

Reply:
Common practice appears to be an `IllegalArgumentException`